### PR TITLE
add missing quotes to target property hints values

### PR DESCRIPTION
**Pull request by DaHaiz**

### CODE REVIEW

File: content/en/guides/get-started/commands.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ npm run start
 
 Nuxt.js gives you the ability to host your web application on any static hosting.
 
-To deploy a static generated site make sure you have `target: static` in your `nuxt.config.js`.(For Nuxt >= 2.13:)
+To deploy a static generated site make sure you have `target: 'static'` in your `nuxt.config.js`.(For Nuxt >= 2.13:)
```
Comment (Atinux) [inline content/en/guides/get-started/commands.md:120]: I think this wording should be fix: `(For Nuxt >= 2.13:)`

Comment (DaHaiz) [reply content/en/guides/get-started/commands.md:120]: you mean removing the colon? didnt notice that in the first place. can add it to my PR if you like.

Comment (Atinux) [reply content/en/guides/get-started/commands.md:120]: Would be nice :)

Comment (DaHaiz) [reply content/en/guides/get-started/commands.md]: done! :) and thx for building nuxt!!

**Review by danielroe — APPROVED** [review]

File: content/en/guides/get-started/commands.md [hunk 2]
```diff
@@ -117,7 +117,7 @@ npm run start
 
 Nuxt.js gives you the ability to host your web application on any static hosting.
 
-To deploy a static generated site make sure you have `target: static` in your `nuxt.config.js`.(For Nuxt >= 2.13:)
+To deploy a static generated site make sure you have `target: 'static'` in your `nuxt.config.js`.(For Nuxt >= 2.13)
```
[the remark this answers is not in the document]
Comment (Atinux) [reply content/en/guides/get-started/commands.md]: Is it good enough?

[the remark this answers is not in the document]
Comment (danielroe) [reply content/en/guides/get-started/commands.md]: I think this comment managed not to be posted until after you already made the fix - which is perfect 💯

**Pull request merged**